### ISSUE TITLE
Upgrade to Elasticsearch 8 because 7 reaches EOL on 10.02.2023

### ISSUE THREAD

**Issue by SunnyPaprika** [nextcloud/fulltextsearch_elasticsearch#240]
Elasticsearch 7 reaches EOL next month. fulltextsearch_elasticsearch does not work with Elasticsearch 8.

**Comment by derritter88** [comment]
+1

**Comment by brendan-pike** [comment]
I'm needing to install elasticsearch for our company but I'm confused as to what to do, I'm hesitant to install v7 if its about to be unsupported.  Can we have a status update please.

**Comment by ne0YT** [comment]
the wiki even tells one to install v6..

**Comment by brendan-pike** [comment]
It seems to me that ReadonlyREST is also no longer required for authentication. If so it would be also good for the wiki to be updated to show how to setup authentication with elasticsearch inbuilt security now.

**Comment by derritter88** [comment]
I did the step and upgrade to the current EL 8.x version.
What is interessting: Finding already indexed files works without any issues and also the fulltextsearch:test command works.

**Comment by arnowelzel** [comment]
No, Elasticsearch 7.16.x will be supported until 2023-06-07 and Elasticsearch 7.17.x will be supported even longer until Elasticsearch 9 is released.

Also see here: https://www.elastic.co/support/eol

**Comment by madbob** [comment]
fulltextsearch_elasticsearch supports ElasticSearch 8 since #441, backported to v31 of the app.
This issue can be closed.
